src/components/MDXComponents.tsx

```diff
@@ -52,65 +31,57 @@ const MDXComponents = {
   // headings
   h1: ({ children }: any) => {
     return (
-      <Heading as='h1' textAlign='start' fontSize='4xl' mb={5}>
+      <Heading as='h1' textAlign='start' mb={5} {...header1}>
```

Comment:
`h1`[font size should be 44px](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=694%3A2580&t=hfe4htIHQvUUNT87-0)

Reply:
<img width="907" alt="image" src="https://user-images.githubusercontent.com/54227730/204351790-6de74394-9609-4cdc-ba7e-79de5534514a.png">
h1 is only 44px on the mobile homepage.

According to design system we should have 34px (2.125rem) here.

With Chakra theme defaults: `--chakra-fontSizes-4xl: 2.25rem` so it actually looks like the `4xl` variable is 2px too big. I could add a foundations override to define `4xl` as `2.125rem` instead, of I could just use `fontSize='2.125rem'` here

Reply:
Also worth noting, we should never actually be using this... In the [slug] template file we render the `title` from each markdown page using a declared h1 within that template. The `h1` here would only render if we use a `# Header` within a markdown file, which we should never do.

We should consider simply removing this from the `MDXComponents` file.

Reply:
exporting from figma to sass, that's why it's in px and not rem:

desktop
@mixin text-style-final-h-1 {font-size:34px;}
@mixin text-style-final-h-2 {font-size:28px;}
@mixin text-style-final-h-3 {font-size:22px;}
@mixin text-style-final-h-4 {font-size:18px;}
@mixin text-style-final-h-5 {font-size:16px;}
@mixin text-style-final-h-6 {font-size:14px;}

mobile
@mixin text-style-final-mobile-h-1 {font-size:30px;}
@mixin text-style-final-mobile-h-2 {font-size:24px;}
@mixin text-style-final-mobile-h-3 {font-size:22px;}
@mixin text-style-final-mobile-h-4 {font-size:16px;}
@mixin text-style-final-mobile-h-5 {font-size:16px;}
@mixin text-style-final-mobile-h-6 {font-size:14px;}

Does this helps?

Reply:
@nhsz I fixed the with @wackerow on a call

Reply:
👍 
Yes, per our discussion, updated the mobile size for h1/h2/h3 to be 30px (1.875rem), 24px (1.5rem) and 20px (1.25rem). Mobile h4/h5/h6 will be kept the same as the existing desktop sizes.

Desktop sizes match the above comment.

Reply:
cc: @nhsz